### ISSUE TITLE
LDAP setup fails

### ISSUE THREAD

**Issue by Artim96** [Icinga/icingaweb2#5124]
## Describe the bug
After being told [here](https://community.icinga.com/t/ldap-setup-error/12658) that this seems to be an actual bug, I'm reporting it here.

I can do the setup routine for setting up LDAP authentication right after the group settings page. Then it will fail with this error screen:
`Call to undefined method Icinga\Repository\RepositoryQuery::setUsePagedResults()`

```
#0 /usr/share/icingaweb2/modules/setup/application/forms/AdminAccountPage.php(110): Icinga\Module\Setup\Forms\AdminAccountPage->fetchGroups()
#1 /usr/share/php/Icinga/Web/Form.php(776): Icinga\Module\Setup\Forms\AdminAccountPage->createElements()
#2 /usr/share/php/Icinga/Web/Wizard.php(706): Icinga\Web\Form->create()
#3 /usr/share/icingaweb2/modules/setup/application/views/scripts/index/parts/wizard.phtml(1): Icinga\Web\Wizard->getForm()
#4 /usr/share/php/Icinga/Web/View.php(235): include(String)
#5 /usr/share/icinga-php/vendor/vendor/shardj/zf1-future/library/Zend/View/Abstract.php(886): Icinga\Web\View->_run()
#6 /usr/share/icingaweb2/modules/setup/application/views/scripts/index/index.phtml(210): Zend_View_Abstract->render()
#7 /usr/share/php/Icinga/Web/View.php(235): include(String)
#8 /usr/share/icinga-php/vendor/vendor/shardj/zf1-future/library/Zend/View/Abstract.php(886): Icinga\Web\View->_run()
#9 /usr/share/icinga-php/vendor/vendor/shardj/zf1-future/library/Zend/Controller/Action/Helper/ViewRenderer.php(910): Zend_View_Abstract->render()
#10 /usr/share/icinga-php/vendor/vendor/shardj/zf1-future/library/Zend/Controller/Action/Helper/ViewRenderer.php(931): Zend_Controller_Action_Helper_ViewRenderer->renderScript()
#11 /usr/share/icinga-php/vendor/vendor/shardj/zf1-future/library/Zend/Controller/Action/Helper/ViewRenderer.php(970): Zend_Controller_Action_Helper_ViewRenderer->render()
#12 /usr/share/icinga-php/vendor/vendor/shardj/zf1-future/library/Zend/Controller/Action/HelperBroker.php(277): Zend_Controller_Action_Helper_ViewRenderer->postDispatch()
#13 /usr/share/icinga-php/vendor/vendor/shardj/zf1-future/library/Zend/Controller/Action.php(527): Zend_Controller_Action_HelperBroker->notifyPostDispatch()
#14 /usr/share/php/Icinga/Web/Controller/Dispatcher.php(76): Zend_Controller_Action->dispatch()
#15 /usr/share/icinga-php/vendor/vendor/shardj/zf1-future/library/Zend/Controller/Front.php(954): Icinga\Web\Controller\Dispatcher->dispatch()
#16 /usr/share/php/Icinga/Application/Web.php(294): Zend_Controller_Front->dispatch()
#17 /usr/share/php/Icinga/Application/webrouter.php(105): Icinga\Application\Web->dispatch()
#18 /usr/share/icingaweb2/public/index.php(4): require_once(String)
#19 {main}
```
The steps before validate, but that last step doesn't have a validate button. After pressing next, the error appears reproducibly.

## Your Environment
Include as many relevant details about the environment you experienced the problem in

* Icinga Web 2 version: 2.12.0
* Used modules and their versions (System - About): n/a
* Web browser used: Google Chrome, Firefox
* Icinga 2 version used (icinga2 --version): 2.14.0
* PHP version used (php --version): 8.2.7
* Server operating system and version: Debian Testing

**Comment by juarezjx** [comment]
When will 2.12.1 be available?  I do not see it in the Ubuntu repo or in this github

**Comment by Artim96** [comment]
Time will tell. You can track the progress in the milestones: [https://github.com/Icinga/icingaweb2/milestones](https://github.com/Icinga/icingaweb2/milestones) 

It has been at 50 % for a few weeks now and it's now down to 44 %. So it seems it will be some months to go...

**Comment by nilmerg** [comment]
Months to go for a minor release :rofl: No, I don't promise anything, but that's a _bit_ exaggerated.

**Comment by Artim96** [comment]
Well, that's what current progress suggests. It's been a month that this has been marked as to be fixed in 2.12.1, I don't know how long this milestone has already been open, but progress suggests it will be at least another month to go.

**Comment by slalomsk8er** [comment]
This kind of estimation is quite naive: https://xkcd.com/612/
